src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -94,11 +95,23 @@ public function getProxyFactoryCode(Definition $definition, $id, $factoryCode =
      */
     public function getProxyCode(Definition $definition)
     {
-        return preg_replace(
+        $code = $this->classGenerator->generate($this->generateProxyClass($definition));
+
+        $code = preg_replace(
             '/(\$this->initializer[0-9a-f]++) && \1->__invoke\(\$this->(valueHolder[0-9a-f]++), (.*?), \1\);/',
             '$1 && ($1->__invoke(\$$2, $3, $1) || 1) && $this->$2 = \$$2;',
-            $this->classGenerator->generate($this->generateProxyClass($definition))
+            $code
         );
+
+        if (version_compare(Version::getVersion(), '2.2', '<')) {
```

Comment:
the class doesn't exist in the lowest versions we support, per composer.json

Reply:
and so should be changed to `!\class_exists(Version::class) || version_compare(Version::getVersion(), '2.2', '<')`

Reply:
Good catch!